src/main.rs

```diff
@@ -14,10 +14,9 @@ fn main() {
     let mut buffer = vec![];
     let mut cursorpos:usize = 0;
     const SPALTEN:usize = 16;
-    let mut screenheight;
 
     initscr(); //start ncursesw
-    screenheight = to_usize(getmaxy(stdscr));
+    let screenheight : usize = getmaxy(stdscr) as usize;
```

Comment:
An underflow: http://is.gd/sUjkFO

I'm not really a big fan of casting in Rust, even though the behavior is clearly defined :)